code/rust-sokoban-c03-05/src/main.rs

```diff
@@ -44,9 +45,17 @@ impl event::EventHandler for Game {
             time.delta += timer::delta(context);
         }
 
-        // Run event system
+        // Run event_handler system
```

Comment:
small nit, we could use slightly shorter names like: `box_placed_on_spot::EventHandlerSystem {};`, same for the other 2.

Reply:
i know just renaming the import isn't what you were asking. i just threw it in to see what it looked like without really changing anything